Web/Controller/ImageStyle.hs

```diff
@@ -0,0 +1,45 @@
+module Web.Controller.ImageStyle where
+
+import Web.Controller.Prelude
+import IHP.ControllerSupport
+import System.Directory (doesFileExist)
+import qualified Data.Text as T
+
+
+instance Controller ImageStyleController where
+    action RenderImageStyleAction { width, height, originalImagePath } = do
+        -- Get the original image directory and UUID from the path.
+        let (originalImageDirectory, uuid) = extractDirectoryAndUUID originalImagePath
+
+        let size = show width <> "x" <> show height
+        let imageStylePathDirectory = originalImageDirectory <> "/imageStyles/" <> size
+        let imageStylePath = imageStylePathDirectory <> "/" <> uuid
+
+        -- @todo: How to get rid of the "static/" prefix?
```

Comment:
Should we expose [storage](https://github.com/digitallyinduced/ihp/blob/248cb16b361adb4e8838b33a496f7c2be09541de/IHP/FileStorage/ControllerFunctions.hs#L363-L364) on IHP core?

Reply:
👍